src/core/webidl-index.js

```diff
@@ -63,5 +63,8 @@ export function run() {
     });
   // Remove duplicate IDs
   pre.querySelectorAll("*[id]").forEach(elem => elem.removeAttribute("id"));
+  pre
+    .querySelectorAll(".idlHeader:not(:first-child)")
```

Comment:
Also, not removing the `:first-child` means the button will link to first interface only (which doesn't make sense).

Reply:
Lol, woops! Good point. I think i need to add a function to add these headers as needed.

Reply:
How about we do something like:
``` js
document.querySelectorAll("pre.webidl :not(.idlHeader), pre.idl :not(.idlHeader)")
  .forEach(idls => idlIndex.append(idls));
```

Reply:
Went with the function instead. It was a bit simpler.